pkg/provisioner/ironic/dependencies.go

```diff
@@ -36,14 +35,15 @@ func (p *ironicProvisioner) checkIronicConductor() (ready bool, err error) {
 	}
 
 	driverCount := 0
-	pager.EachPage(p.ctx, func(_ context.Context, page pagination.Page) (bool, error) {
+	_ = pager.EachPage(p.ctx, func(_ context.Context, page pagination.Page) (bool, error) {
```

Comment:
Is this an error? We probably shouldn't just ignore it.

Reply:
Yes, this is an error and it breaks the test

Reply:
@dtantsur please check the unit test failure now caused by this, I would need help understanding this in case we dont want to ignore it.